util/strings.go

```diff
@@ -59,3 +59,25 @@ func EscapeJQLReservedChars(in string) string {
 	in = strings.Replace(in, `:`, `\\:`, -1)
 	return in
 }
+
+// GetSetDifference returns the elements in A that are not in B
+func GetSetDifference(a, b []string) (difference []string) {
```

Comment:
avoid declaring a variable for the return argument in the signature.

Reply:
Oops, I originally had this as a naked return.  Removing.